src/wait-strategy.ts

```diff
@@ -64,3 +73,31 @@ export class HostPortWaitStrategy extends AbstractWaitStrategy {
     );
   }
 }
+
+export type Log = string;
+
+export class LogWaitStrategy extends AbstractWaitStrategy {
+  constructor(private readonly message: Log) {
+    super();
+  }
+
+  public async waitUntilReady(container: Container): Promise<void> {
+    return new Promise(async (resolve, reject) => {
+      const stream = await container.logs();
+      stream
+        .on("data", chunk => {
```

Comment:
Is this chunk a single line?
If this is not line-buffered, it might be possible to miss certain messages and wait forever.

Reply:
It is now line buffered 👍